.pre-commit-config.yaml

```diff
@@ -49,3 +48,8 @@ repos:
     hooks:
       - id: gitlint
         stages: [commit-msg]
+  - repo: https://github.com/adrienverge/yamllint.git
+    rev: v1.30.0
+    hooks:
+      - id: yamllint
+        args: [--strict, -c=.yamllint]
```

Comment:
Could this be moved to run before commit-msg stage (above gitlint)? Otherwise pre-commit output looks a bit confusing.